static/js/shared/url_utils.ts

```diff
@@ -15,15 +15,9 @@
  */
 
 /**
- * Constants and functions shared across components.
+ * URL related helpers shared across components.
```

Comment:
let's put this in the utils folder: https://github.com/datacommonsorg/website/tree/master/static/js/utils

Reply:
Ah ok